CHANGES.md

```diff
@@ -30,6 +30,12 @@ OpenSSL 3.0
 
    *Shane Lontis*
 
+ * The SSL option SSL_OP_IGNORE_UNEXPECTED_EOF is introduced. If that option
+   is set, the unexpected EOF is ignored, it pretends close notify was received
+   instead and so returned error becomes SSL_ERROR_ZERO_RETURN.
```

Comment:
I think it should be "an unexpected EOF", "a close notify", and "the returned error".

Reply:
Done. Thanks!